dist/index.html

```diff
@@ -28,79 +28,77 @@
         <span class="menu-btn__burger"></span>
```

Comment:
- [ ] The index page in mobile view should have a white background according to the given design, kindly address this issue.

Given Design
![image](https://user-images.githubusercontent.com/81932478/169594693-c35ab869-77e5-4830-a67d-d5e07fcbca40.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/169594737-814c86a3-8b04-4097-b871-7a05ab1823af.png)